GrandCentralBoard/Widgets/Bonus/Scene/BubbleScalingController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  BubbleScalingController.swift
+//  GrandCentralBoard
+//
+//  Created by Bartłomiej Chlebek on 07/04/16.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import SpriteKit
+
+protocol BubbleScalingControllerDelegate: class {
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeUp spriteNode: SKSpriteNode)
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeDown spriteNode: SKSpriteNode)
+}
+
+extension BubbleScalingControllerDelegate {
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeUp spriteNode: SKSpriteNode) { }
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeDown spriteNode: SKSpriteNode) { }
```

Comment:
Yea, I wasn't sure which way to go. I picked the one with a more complete API. But I suppose the less the better.
I'll remove `didScaleSpriteNodeUp` 👍